src/AzureStack/UpdateAdmin/Update.Admin/Microsoft.AzureStack.Management.Update.Admin.csproj

```diff
@@ -0,0 +1,20 @@
+﻿
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.AzureStack.Management.Update.Admin</PackageId>
+    <Description>Provides developers with libraries for the Update admin under AzureStack to manage plans and stuff.</Description>
+    <VersionPrefix>0.1.0-preview</VersionPrefix>
+    <AssemblyName>Microsoft.AzureStack.Management.Update.Admin</AssemblyName>
+    <PackageTags>Microsoft Azure Stack;Update;REST HTTP client;azureostackfficial;netcore451511</PackageTags>
+    <PackageReleaseNotes>This is a preview release of the AzureStack Update Admin SDK.</PackageReleaseNotes>
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Update="node_modules\JSONSelect\site\css\droid_sans.tt">
```

Comment:
@deathly809 what are these files?

Reply:
I have no clue,  someone else was working on this.  I will clean it up.